### ISSUE TITLE
Widgets (images, others) not properly rendering on page load/reload

### ISSUE THREAD

**Issue by brtwrst** [silverbulletmd/silverbullet#164]
Concerning the attachments I am seeing a weird behavior.

1. When I go to any page, I can not see any pictures/attachments at first.
![image](https://user-images.githubusercontent.com/6873862/205688990-36255d7e-3940-4c4e-bd38-d04ab997d28d.png)

2. I have to find one of the invisible pictures and put the cursor on the line to make the text visible
![image](https://user-images.githubusercontent.com/6873862/205689060-027e4127-0135-46f5-9748-5eb8cc452284.png)

3. If I just move the cursor away nothing will change but if I change/update anything about the link -> all pictures in the document will appear.
![image](https://user-images.githubusercontent.com/6873862/205689538-1af90485-b180-41d3-ab22-bf9763bb4bd7.png)

As soon as I press F5 or otherwise reload the page - all pictures are invisible again

This happens with `--user` and without so it seems to be unrelated to that.
From the Devtools I can see that the pictures are not even downloaded until step 3 described above is executed

**Comment by brtwrst** [comment]
I added a picture to another random page and it works fine there.
Then I went back to my broken page and added a new picture there, this now also works again - miraculously.
But the 4 pictures that were already on the site still experience the problem.

* I have checked if it is because of the fact that the attachments are within the same subfolder as the .md file (due to the "import" from another note app - but it happens regardless of the storage location of the picture. (Aside: Is it normal that attachments always upload to the root /space folder instead of the folder the .md file is actually in?)
* I have also checked if the filename has an effect but it seems like it does not.
* I loaded up yesterdays docker image again - same problem
* I tried another browser - same problem
* Checked it on my mobile - same problem
* I downloaded the png file and uploaded it again to the same page with another name - same problem
